### DISCUSSION TITLE
Add licence terms for icons

### DISCUSSION THREAD

**Discussion by MarcelloNicoletti** [elrumo/macOS_Big_Sur_icons_replacements#1249] [Ideas]
I recently submitted an icon that incorporates the related project's [main logo](https://github.com/nashamri/spacemacs-logo). That logo is shared under the `Creative Commons Attribution-ShareAlike 4.0 International` licence. In order to correctly comply with the licence terms my submission should contain an attribution back to the original author and indicate that it is also `CC BY SA 4.0`. This could just be a text entry on the submission. If there are enough common licenses it could also be a drop down that enables extra information fields if needed.

**Comment by cake03** [comment]
If you have a problem with all the licensing you can use https://www.spacemacs.org/img/logo.svg

**Reply by MarcelloNicoletti** [reply]
That is the one I'm using. The original author shared that with the project under `CC BY SA 4.0`. My suggestion is to add a field to this site for licensing information for cases like mine. For now since I only plan on submitting the one I plan on leaving that info in my bio on the site (once saving is fixed, at least).
